cluster/gce/manifests/cloud-controller-manager.manifest

```diff
@@ -26,7 +26,7 @@
     },
     "command": ["/cloud-controller-manager"],
     "args": [
-      "--log-file=/var/log/cloud-controller-manager.log",
+      "--log_file=/var/log/cloud-controller-manager.log",
```

Comment:
I notice we have a smattering of manifests and the configure-helper.sh script which still set the --log-file flag for other processes. Not saying we should do anything in this PR but we should look at adding some sort of test for this. This flag is set by klog which indicates we can expect updating versions of the other binaries can cause the same problem in them.

Reply:
Thanks for adding the info. Other process with go-runner will expect `--log-file`(e.g.: https://github.com/kubernetes/cloud-provider-gcp/blob/c497d0da8d551515b61fb132612718d390809814/cluster/gce/manifests/kube-controller-manager.manifest#L28).

Reply:
:+1: